src/components/transactionsV2/explorerTransactionsV2/explorerTransactionsV2.js

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+import TransactionsOverviewV2 from '../transactionsOverviewV2';
+import txFilters from '../../../constants/transactionFilters';
+import TransactionsOverviewHeader from '../transactionsOverviewHeader/transactionsOverviewHeader';
+import routes from '../../../constants/routes';
+import TabsContainer from '../../toolbox/tabsContainer/tabsContainer';
+
+class ExplorerTransactions extends React.Component {
+  constructor() {
+    super();
+
+    this.state = {
+      filter: {},
+      customFilters: {},
+    };
+
+    this.saveFilters = this.saveFilters.bind(this);
+    this.clearFilter = this.clearFilter.bind(this);
+    this.clearAllFilters = this.clearAllFilters.bind(this);
+    this.changeFilters = this.changeFilters.bind(this);
+    this.onInit = this.onInit.bind(this);
+    this.onLoadMore = this.onLoadMore.bind(this);
+    this.onFilterSet = this.onFilterSet.bind(this);
+    this.onTransactionRowClick = this.onTransactionRowClick.bind(this);
+    // this.searchMoreVoters = this.searchMoreVoters.bind(this);
```

Comment:
if it is not used this could be remove it

Reply:
This code will be used in a future task, for when we add the votes and delegates tabs on the page.

Reply:
ok perfect, I see the comment now in the code